config/locales/views/autocomplete/en.yml

```diff
@@ -1,4 +1,4 @@
 en:
   views:
     autocomplete:
-      no_matching_repo: "We couldn't find any matching repositories"
+      no_matching_repo: 'We couldn''t find any matching repositories'
```

Comment:
@BelieveC looks like you added a `'` in `couldn''t`. Could you remove that? 😅

Reply:
@d12 , Actually one extra `'` is required as an escape symbol. Therefore I have added one extra `'`.

Reply:
Ohh I see what's happening here. Correct me if I'm wrong, but I think the behavior of `''` in a string is concatenation, since all you're doing is terminating one string and beginning another, and `"a" "b"` in ruby is equal to `"ab"`

```
irb(main):002:0> 'foo''bar'
=> "foobar"
```

If that's the case, `\'` should work.

Reply:
> I think the behavior of '' in a string is concatenation, since all you're doing is terminating one string and beginning another, and "a" "b" in ruby is equal to "ab"

Exactly! This is what I have done there.

> If that's the case, \' should work.

I tried \, but it works within " ". Here it didn't work as we have ' ' .

Reply:
`\'` should work within ''. You need to escape it with a backslash. 

The way you've done it actually loses the `'`.

```
irb(main):004:0> 'We couldn''t find any matching repositories'
=> "We couldnt find any matching repositories"
```

Notice the missing `'` in the string.

Reply:
My Bad! Wait will fix it :+1:

Reply:
@d12 , Actually `'couldn''t'` is parsed as `couldn't` in YAML. I checked it out here http://yaml-online-parser.appspot.com/

Reply:
Lemme know if any other changes are required.

Reply:
Whoops, I forgot this is YAML and not ruby. Also, TIL `''` => `'` in YAML.

Sorry about the hassle!